contributing.md

```diff
@@ -40,16 +41,16 @@ Here's an example using all of the possible `feature.add` options:
 ```ts
 import React from 'dom-chef';
 import select from 'select-dom';
-import delegate, {DelegateSubscription} from 'delegate-it';
+import delegateIt, {DelegateSubscription} from 'delegate-it';
 import features from '../libs/features';
 
-let delegate;
+let delegate: DelegateSubscription | undefined;
 function log() {
 	console.log('✨', <div className="rgh-jsx-element"/>);
 }
 function init(): void {
 	// Events must be set via delegate, unless shortlived
-	delegate = delegate('.btn', 'click', log);
+	delegate = delegateIt('.btn', 'click', log);
```

Comment:
This was fine as `delegate`

Reply:
it conflicted with `let delegate;`

Reply:
Indeed. Better change that variable than this one then. It was just a random example

Reply:
@fregante done!